pkg/cinder/dbsync.go

```diff
@@ -55,16 +57,16 @@ func DbSyncJob(instance *cinderv1beta1.Cinder, labels map[string]string) *batchv
 								RunAsUser: &runAsUser,
 							},
 							Env:          env.MergeEnvs([]corev1.EnvVar{}, envVars),
-							VolumeMounts: GetVolumeMounts(false),
+							VolumeMounts: GetVolumeMounts(false, []cinderv1beta1.CinderExtraVolMounts{}, []storage.PropagationType{storage.DBSync}),
```

Comment:
I think the `[]storage.PropagationType{storage.DBSync})` should be a constant that we only define once, right?

And maybe `[]cinderv1beta1.CinderExtraVolMounts{}` should be a variable in this method that we can use in all 3 instances where it is used.

Reply:
Right, they're basically constants or variables that can be reused all over the code, hence I moved the first to the `const` file, while the second it's just a regular variable that can be reused many times in the same function.